### ISSUE TITLE
Don't work with horizontal scroll

### ISSUE THREAD

**Issue by PubliAlex** [geosigno/simpleParallax.js#118]
Hello,

Just to notice that on website with horizontal scrolling, the parallax effect doesn't work.

Best regards,

**Comment by geosigno** [comment]
Yes you're right. That could be a thing to add, but maybe quite complex. Keep it open for now.
